src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -34493,6 +34493,32 @@ public class C
 ");
             Assert.Null(model.GetOperation(node3).Parent);
         }
+
+        [Fact]
+        public void OutVarInConstructorUsedInInitializer()
+        {
+            var source =
+@"
+public class C
+{
+    public int Number{ get; set; }
+    
+    public C(out int n)
+    {
+        n = 1;
+    }
+
+    public static void Main()
+    {
+        C c = new C(out var i) { Number = i };
+        System.Console.WriteLine(c.Number);
+    }
+}
+";
+            var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.DebugExe, parseOptions: TestOptions.Regular);
```

Comment:
Use the `CreateCompilation` helper. It's the standard helper we should be using whenever possible.

Reply:
Fixed.